tensorboard/plugins/interactive_inference/interactive_inference_plugin.py

```diff
@@ -249,29 +250,38 @@ def _infer(self, request):
       if request.method != 'GET':
         tf.logging.error('%s requests are forbidden.', request.method)
         return http_util.Respond(request, {'error': 'invalid non-GET request'},
-                                 'application/json', code=405)
+                                    'application/json', code=405)
+
+      inference_addresses = request.args.get('inference_address').split(',')
+      model_type = request.args.get('model_type')
+      model_names = request.args.get('model_name').split(',')
+      model_versions = request.args.get('model_version').split(',')
+      model_signatures = request.args.get('model_signature').split(',')
 
-      serving_bundle = inference_utils.ServingBundle(
-          request.args.get('inference_address'),
-          request.args.get('model_name'), request.args.get('model_type'),
-          request.args.get('model_version'),
-          request.args.get('model_signature'),
-          request.args.get('use_predict') == 'true',
-          request.args.get('predict_input_tensor'),
-          request.args.get('predict_output_tensor'))
       indices_to_infer = sorted(self.updated_example_indices)
       examples_to_infer = [self.examples[index] for index in indices_to_infer]
-
-      # Get inference results proto and combine with indices of inferred
-      # examples and respond with this data as json.
-      inference_result_proto = platform_utils.call_servo(
-          examples_to_infer, serving_bundle)
-      new_inferences = inference_utils.wrap_inference_results(
-          inference_result_proto)
-      infer_json = json_format.MessageToJson(
-          new_inferences, including_default_value_fields=True)
-      infer_obj = json.loads(infer_json)
-      resp = {'indices': indices_to_infer, 'results': infer_obj}
+      infer_objs = []
+      for model_num in xrange(len(inference_addresses)):
```

Comment:
what happens if someone puts in 2 inference addresses but only one model name? probably should return an error here. same with other of the args that are split.